controller/responseTypeController.js

```diff
@@ -0,0 +1,39 @@
+const ResponseType = require('../model/responseTypeModel');
+const HttpStatus = require('http-status-codes');
+
+
+exports.getAll = function(req, res) {
```

Comment:
Refactor. We are using ES6 notation throughout thw whole project. Take a look at other controllers and refactor accordingly.